src/main/proto/marathon.proto

```diff
@@ -141,6 +141,23 @@ message MarathonTask {
     repeated string local_volume_ids = 1;
     required State state = 2;
   }
+  enum MarathonTaskStatus {
+    Reserved = 1;
+    Created = 2;
+    Error = 3;
+    Failed = 4;
+    Finished = 5;
+    Killed = 6;
+    Killing = 7;
+    Lost = 8;
+    Running = 9;
+    Staging = 10;
+    Starting = 11;
+    Unreachable = 12;
+    Gone = 13;
+    Unknown = 14;
```

Comment:
Unknown will be an explicit mesos status, therefore I introduced a state Invalid with order number 0 which is mapped to a serialization error.